src/runtime/types/utils.ts

```diff
@@ -24,3 +24,11 @@ export type ArrayOrWrapped<T> = T extends any[] ? T : Array<T>
 export type PartialString<T> = {
   [K in keyof T]?: string
 }
+
+export type SelectItems<T> = T[] | T[][]
```

Comment:
Maybe we should name these types `Combobox` since they are used in components using the Radix Vue Combobox?

Reply:
Yes but here they're also used for the `USelect` component, which isn't based on `ComboBox`, so I thought `Select` is the common denominator for all three